Code/Framework/AzCore/AzCore/Debug/Trace.cpp

```diff
@@ -69,6 +70,10 @@ namespace AZ::Debug
     static AZ::EnvironmentVariable<int> g_assertVerbosityLevel;
     static AZ::EnvironmentVariable<int> g_logVerbosityLevel;
 
+    static const char* assertsAutoBreakUID = "assertsAutoBreak";
```

Comment:
Since these variables are constants they should use the CapitalName scheme and not `s_<name>` scheme as per the [coding standard](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names)

Reply:
Yea I can do that. This is an older file so the conventions are sort of all over the place